schemas/add-to-cart-button.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Add To Cart Button",
+	"markdownDescription": "https://developers.vtex.com/vtex-developer-docs/docs/vtex-add-to-cart-button\n\nThe add-to-cart-button is a block responsible for adding products in the Minicart (minicart.v2).\n\n`\"vtex.add-to-cart-button@0.x\"`",
+	"type": "object",
+	"properties": {
+		"props": {
+			"type": "object",
+			"description": "Block's properties",
+			"properties": {
+				"blockClass": { "$ref": "./shared/blockClass.schema.json" },
+				"onClickBehavior": {
+					"type": "string",
+					"oneOf": [
+						{ "const": "go-to-product-page" },
+						{ "const": "add-to-cart" },
+						{ "const": "ensure-sku-selection" }
+					],
```

Comment:
```suggestion
					"oneOf": [
						{ "const": "go-to-product-page" },
						{ "const": "add-to-cart" },
						{ "const": "ensure-sku-selection" }
					],
```
```suggestion
					"oneOf": [
						{ "const": "go-to-product-page" },
						{ "const": "add-to-cart" },
						{ "const": "ensure-sku-selection", "markdownDescription": "if multiple SKUs are available, users will be redirected to the product page to select the desired one.\n If the product only has 1 SKU available, it will be added to the cart once the button is clicked on" }
					],
```